development/client/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "colors",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@apollo/client": "^3.0.0-rc.4",
```

Comment:
Maybe just replace `^3.0.0-rc.4` with `latest`, as we've already jumped up to `3.0.0-rc.10` in a few days, and there are more coming! 😂

Reply:
🚀 Good idea! Done!